### DISCUSSION TITLE
Error Accessing GitHub Education Community Exchange & Benefits Not Received

### DISCUSSION THREAD

**Discussion by ghost** [community/community#147606] [GitHub Education]
### Hi everyone,

Hi everyone,

I recently got my GitHub Education Student Developer Pack approved, and according to the FAQ, I should have received instant access to the Campus resources, Community Exchange

 However, I’m facing the following issues:

1. Community Exchange Access Error: When I try to access the Community Exchange, I get a "403 Access Denied" error.

2. Benefits Not Activated: Even after almost 48 hours of approval, my benefits are still not visible or accessible.

Has anyone else faced similar issues? Is this a delay in syncing my account, or is there something additional I need to do?
Issue screenshots
![Screenshot_20241220-183621](https://github.com/user-attachments/assets/be0f6df0-7995-4a26-ace9-dc259f83a939)
![Screenshot_20241220-182650](https://github.com/user-attachments/assets/935aca9d-ae2c-42e1-a030-7ab91855d0c4)

**Comment by davevad93** [comment]
Hi @Sanyam4007 ,

You've been approved but your benefits are still not available. 
when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Sanyam4007 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by ghost** [reply]
Hey @davevad93 Thanks a lot for response it was approved today but there seems to be a issue with azure I am unable to claim it it doesn't verify my student status for some reason

**Reply by davevad93** [reply]
Hey @Sanyam4007 , this is a known issue that has been solved here #135739 , you have the following options:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I recommend the first one.

**Reply by ghost** [reply]
@davevad93 Thank you for helping

**Reply by davevad93** [reply]
You're welcome @Sanyam4007 👍
